lib/net.js

```diff
@@ -825,7 +825,17 @@ protoGetter('bytesWritten', function bytesWritten() {
       bytes += Buffer.byteLength(el.chunk, el.encoding);
   });
 
-  if (data) {
+  if (Array.isArray(data)) {
+    // was a writev, iterate over chunks to get total length
+    for (var i = 0; i < data.length; i++) {
+      const chunk = data[i];
+
+      if (chunk instanceof Buffer)
```

Comment:
I think @mscdex’s suggestion makes sense. Also, using `process.binding('util').isUint8Array(chunk)` should be a faster alternative to a full `instanceof` check, that should be okay here.